cmd/config.go

```diff
@@ -19,20 +19,27 @@ package cmd
 import (
 	"os"
```

Comment:
Can your commit be separated into new lines? following an 80 column rule. 

It gets harder to read 'git logs'